packages/developer-portal/src/components/pages/app-detail/app-authentication-detail/__styles__/app-authentication-detail.ts

```diff
@@ -0,0 +1,53 @@
+import { css } from 'linaria'
+import { grey } from '@/core/__styles__/colors'
+
+export const authenticationCodeWrap = css`
+  padding: 8px;
+  border: 1px solid ${grey};
+  min-width: 80%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`
+
+export const authenticationCode = css`
+  word-break: break-all;
+  margin-bottom: 0 !important;
```

Comment:
Can we remove !important ?

Reply:
Hi @tanphamhaiduong ,
I'll fix it in the next PR